controllers/suite_test.go

```diff
@@ -336,15 +336,15 @@ func setUpTestProxy() {
 	}
 
 	err = k8sClient.Create(ctx, &testProxyService)
-	Expect(err != nil || errors.IsConflict(err))
+	Expect(err).NotTo(HaveOccurred())
 
 	testProxyIngress := ingresses.NewIngressForCRWithSSL(
 		nil, types.NamespacedName{Name: testProxyName, Namespace: testProxyNamespace},
 		map[string]string{}, testProxyName+"-dep-svc", strconv.FormatInt(int64(testProxyPort), 10),
 		true, "", testProxyHost, isOpenshift)
 
 	err = k8sClient.Create(ctx, testProxyIngress)
-	Expect(err != nil || errors.IsConflict(err))
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
The test-suite can skip error if the test proxy ingress already exist:
```suggestion
	Expect(err == nil || errors.IsConflict(err))
```